### ISSUE TITLE
Define notification regarding the limited amount of overlays per case.

### ISSUE THREAD

**Issue by carmelorr** [AnSyn/ansyn#920]

**Comment by carmelorr** [comment]
![10 8](https://user-images.githubusercontent.com/26301386/33542290-1c42072a-d8db-11e7-9e9e-86246642ed89.png)
![10 8 1](https://user-images.githubusercontent.com/26301386/33542294-1e0d05be-d8db-11e7-9338-ee5753e34a21.png)
Or "shorten time range for precise results"

**Comment by idanbarak** [comment]
why not show indication in the same location of the "no overlays available" indication?

**Comment by carmelorr** [comment]
@idanbarak you are right. 
![10 8 3](https://user-images.githubusercontent.com/26301386/33612478-501e157a-d9da-11e7-8f01-aa7b190ca489.png)
![10 8 4](https://user-images.githubusercontent.com/26301386/33612499-649c61a0-d9da-11e7-9364-e7f105c430d4.png)
![10 8 4 1](https://user-images.githubusercontent.com/26301386/33612530-774f846c-d9da-11e7-8bee-3398babc5bea.png)

**Comment by carmelorr** [comment]
As discussed today,

**Comment by carmelorr** [comment]
X overlays match your search. Latest 500 are presented, you can change here the time range for better results 

X overlays match your search. Latest 500 are presented, you can change here the time range for precise results 


X overlays match your search. Note that only the latest 500 are presented, you can see more results by clicking here to change the time range


X overlays match your search. Note that only the latest 500 are presented, you can accurate your results by clicking here to change the time range

X overlays match your search. Note that only the latest 500 are presented, you can change the results by clicking here to change the time range

**Comment by carmelorr** [comment]
@idanbarak what do you think is best?

**Comment by idanbarak** [comment]
X overlays match your search. Note that only the latest 500 are presented, click here to edit search time range.

**Comment by carmelorr** [comment]
![10 8 4 4](https://user-images.githubusercontent.com/26301386/33939040-50de433e-e012-11e7-98e9-8ffbe10030eb.png)

**Comment by carmelorr** [comment]
Clicking "here" will open the time picker

**Comment by yuvalbl** [comment]
also treated in PR #972 :
1. duplicated overlays (same id) removed
2. overlay-config token was added back to code
3. additional testing

open:
limit-message is currently by toast (limited number of overlay may not be accurate)

**Comment by carmelorr** [comment]
@yuvalbl please see-
![10 8 4](https://user-images.githubusercontent.com/26301386/34486031-7166401a-efd6-11e7-9703-df8500d31eb0.png)
it is also apply for #921

**Comment by yuvalbl** [comment]
@carmelorr looks good.
another question is - how should we treat cases when it comes to Favorites (which will be displayed even when not matching the case)?

I can display the no. of overlays matching the search, regardless of if it a "favorite" or not. However, in cases there are Favorites which do not match, they will we displayed anyway, so actual number (number of drops) can be different than displayed number in text ("x overlays match your case")
is that OK?

**Comment by carmelorr** [comment]
yes that's OK since the notification is about the query (regardless of the favorites).

**Comment by carmelorr** [comment]
@asafMasa as discussed in person, please add the notification to the time line, above the overlays (favorites). 
please see- 
![10 8 5](https://user-images.githubusercontent.com/26301386/35233491-fe961390-ffa6-11e7-8f83-138038c92189.png)
![10 8 4 2](https://user-images.githubusercontent.com/26301386/35233492-ff8967e8-ffa6-11e7-8478-97e7299b9847.png)

if needed see css- 
/* Rectangle */

position: absolute;
width: 410px;
height: 30px;
left: 715px;
top: 36px;

background: rgba(255, 255, 255, 0.7);
border: 1px solid #BDBDBD;
box-sizing: border-box;
filter: blur(8px);
backdrop-filter: blur(4px);
/* Note: backdrop-filter has minimal browser support */
